lib/models/profile.dart

```diff
@@ -1,29 +1,40 @@
 import 'package:givt_app_kids/models/monsters.dart';
 
-class Profile {
+class Profile implements Comparable {
```

Comment:
Isn't easier to use [Equatable](https://pub.dev/packages/equatable) instead of implementing comparable? It's cleaner and it's the most used way of comparing objects in Flutter. Just want to start a discussion around this.